src/themes/light.scss

```diff
@@ -6,7 +6,7 @@
 
   --wxc-banner-background: #{$wxc-gradation-light-transparent};
   --wxc-banner-text-color: #{$wxc-black-alpha-95};
-  --wxc-video-background: #{$md-white};
+  --wxc-video-background: #{$wxc-white-alpha-100};
```

Comment:
please put this just after --wxc-text-color, it's a generic variable not connected to any specific generic component